Homework_33/script.js

```diff
@@ -0,0 +1,27 @@
+document.addEventListener('DOMContentLoaded', function () {
+
+    let arr = new Array(1, 2, 3);
+
+    Array.prototype.pow = function (n) {
+        let array = this;
+        let copyArray = new Array();
+        for (let i = 0; i < array.length; i++) {
+            copyArray[i] = array[i] ** n;
+        }
+        return copyArray;
+    }
+
+    Function.prototype.defer = function (ms) {
+        let func = this;
+        setTimeout(function () {
+            func.apply(null, [ms]);
+        }, ms);
+    }
+
+    function a() {
+        alert("test");
+    }
+
+    a.defer(5000);
```

Comment:
it could the only line inside of block `DOMContentLoaded`
don't make declaration of functions and variables in this block

Reply:
Understood